modules/openshift-developer-cli-deleting-an-application.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+// * cli_reference/openshift_developer_cli/creating-an-application-with-a-database.adoc
+// * cli_reference/openshift_developer_cli/creating-a-multicomponent-application-with-odo.adoc
+
+[id="deleting-an-application_{context}"]
+
+= Deleting an Application
+
+[IMPORTANT]
+====
+Deleting the application will delete all components associated with the application.
+====
+
+.Procedure
+
+. List the applications in the current project:
++
+----
+$ odo app list
+----
+
+. List the components associated with the applications:
```

Comment:
This step is unnecessary. Moreover you have already mentioned that ```Deleting the application will delete all components associated with the application.```, so you can remove this step.

Reply:
hm, makes sense, but what if the user does not want those components removed by mistake? Just a fail-safe, maybe.

Reply:
as per our convo in slack @amitkrout let's leave this here as a fail-safe for users not really reading the warnings (e.g. me). Just re-assure users that the command doesn't do anything besides listing the components

Reply:
On the other hand, let's just say the app deletes the components the second time, as lists do usually what lists do -> list :)